buildingmotif/ingresses/bacnet.py

```diff
@@ -42,17 +42,22 @@ def __init__(self, ip: Optional[str] = None):
         # for each discovered Device, create a BAC0.device object
         # This will read the BACnet objects off of the Device.
         # Save the BACnet objects in the objects dictionary
-        assert self.network.discoveredDevices is not None
-        for (address, device_id) in self.network.discoveredDevices:  # type: ignore
-            # set poll to 0 to avoid reading the points regularly
-            dev = BAC0.device(address, device_id, self.network, poll=0)
-            self.devices.append(dev)
-            self.objects[(address, device_id)] = []
+        try:
+            assert self.network.discoveredDevices is not None
```

Comment:
Can we make this an Exception rather than an assertion? That way it is more consistent with the use of exceptions/assertions elsewhere in the code (I'm still reminding myself to do this too...)

Reply:
Do we even want this to be an exception? if there aren't any discovered devices maybe we should just give a warning.

Reply:
Yeah, that works. Probably better too to not interrupt operation just to throw an exception